packages/functional-tests/src/server.ts

```diff
@@ -15,6 +15,7 @@ process.on('uncaughtException', (err) => console.log('uncaughtException', err));
 process.on('unhandledRejection', (reason) => console.log('unhandledRejection', reason));
 
 MRE.log.enable('app');
+MRE.log.enable('client-trace');

```

Comment:
>client-trace [](start = 16, length = 12)

What about just `client`? client-trace seems overly long, and `trace` might be misleading/open to misinterpretation.